From 47604609a5d0d9d0161127438b7ad1b8c7ca920b Mon Sep 17 00:00:00 2001
From: Apaczer <94932128+Apaczer@users.noreply.github.com>
Date: Sun, 15 Feb 2026 21:05:40 +0100
Subject: [PATCH] config.def.h: http:/buildbot (uClibc) & org.(musl)

- as upstream uses legacy steward's toolchain with uClibc
- for modern we use our own builds from MiyooCFW/cores nightly
---
 config.def.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/config.def.h b/config.def.h
index 066ae1f57c..abbe0d83a2 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1784,6 +1784,12 @@
 #define DEFAULT_BUILDBOT_SERVER_URL "http://hakchicloud.com/Libretro_Cores/"
 #elif defined(WEBOS)
 #define DEFAULT_BUILDBOT_SERVER_URL "http://retroarch-cores.webosbrew.org/armv7a/"
+#elif defined(MIYOO)
+#if defined(__UCLIBC__)
+#define DEFAULT_BUILDBOT_SERVER_URL "http://buildbot.libretro.com/nightly/dingux/miyoo-arm32/latest/"
+#else
+#define DEFAULT_BUILDBOT_SERVER_URL "http://raw.githubusercontent.com/MiyooCFW/cores/master/cores/musl/latest/"
+#endif
 #elif defined(ANDROID)
 #if defined(ANDROID_ARM_V7)
 #define DEFAULT_BUILDBOT_SERVER_URL "http://buildbot.libretro.com/nightly/android/latest/armeabi-v7a/"
-- 
2.45.2.windows.1

